keyboards/qwertykeys/qk100/info.json

```
@@ -0,0 +1,40 @@
+{
+    "manufacturer": "Qwertykeys",
+    "url": "",
+    "maintainer": "Qwertykeys",
+    "usb": {
+        "vid": "0x4F53"
+    },
+    "matrix_pins": {
+        "cols": ["A7", "B0", "B1", "B2", "B10", "B11", "B12", "B13", "B14", "B15", "A8", "A9", "A10", "A15", "B4", "B5", "B8", "B9", "C13" ],
+        "rows": ["A1", "A2", "A3", "A5", "A4", "A6" ]
+    },
+    "diode_direction": "COL2ROW",
+    "features": {
+        "bootmagic": true,
+        "extrakey": true,
+        "mousekey": true,
+	    "rgblight": true,
+        "nkro": true
+    },
+    "processor": "STM32F103",
+    "board": "STM32_F103_STM32DUINO",
```

This should set most of the bootloader config, minus the custom ld script and dfu suffix/args
```suggestion
    "bootloader": "stm32duino",
```